### PR TITLE
Smoke test

### PR DESCRIPTION
**Pull request by Br4nd0R**
* Added a new GitHub workflow triggered by a release to production.
* New settings file for setting all IPs as trusted on the scratch org.

### CODE REVIEW

File: .github/workflows/smoke_test.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          ./sfdx/install
+          echo $SFDX_HUB_KEY_BASE64 | base64 --decode > sfdx.key
+          sfdx force:auth:jwt:grant --clientid $SFDX_CLIENT_ID --jwtkeyfile sfdx.key --username $SFDX_HUB_USERNAME --setdefaultdevhubusername -a hub
+        env:
+          SFDX_HUB_KEY_BASE64: ${{ secrets.SFDX_HUB_KEY_BASE64 }}
+          SFDX_CLIENT_ID: ${{ secrets.SFDX_CLIENT_ID }}
+          SFDX_HUB_USERNAME: ${{ secrets.SFDX_HUB_USERNAME }}
+      - name: Install CumulusCI
+        run: |
+          pip install cumulusci
```
Comment (davisagli) [inline .github/workflows/smoke_test.yml:33]: fwiw, the command can go on the same line without the pipe when there's a single command.

File: .github/workflows/smoke_test.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      - name: Create test org
+        run: |
+          $(which cci) org info enterprise
```
Comment (davisagli) [inline .github/workflows/smoke_test.yml:36]: I'm pretty sure you can just call cci. The `$(which cci)` idiom was for when cci was being run by coverage.py which needed the full path.

**Review by davisagli — CHANGES_REQUESTED** [review]

File: .heroku/release.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+curl -X POST \
+
+     -H "Accept: application/vnd.github.v3+json" \
+     -H "Authorization: Bearer $token" \
+
+     https://api.github.com/repos/$repo/dispatches \
+
+     -d '{"event_type":"production-release"}'
+
+echo "Done."
```
[the remark this answers is not in the document]
Comment (davisagli) [reply .heroku/release.sh:19]: Also, this dispatch to github needs to be something that's off by default but can be turned on with a config var. We don't want other instances of MetaDeploy triggering our build (and their GITHUB_TOKEN probably doesn't have access to)

[the remark this answers is not in the document]
Comment (Br4nd0R) [reply .heroku/release.sh:19]: I'll start with a 5 minute delay before the robot tests start with.

[the remark this answers is not in the document]
Comment (davisagli) [reply .heroku/release.sh:18]: @Br4nd0R Still needs a way for us to enable it only for some deployments of metadeploy.

**Review by davisagli — APPROVED** [review]
Looking good!

**Pull request merged**